src/client/contextmenu.js

```diff
@@ -68,11 +68,14 @@ export default class ContextMenu {
       }, "CMD+K"],
       ["Browser",     (evt) => {
         this.openComponentInWindow("lively-container", evt).then(comp => {
-          var container = _.last(document.querySelectorAll("lively-container"));
+          let containers = Array.from(document.querySelectorAll('lively-container'))
```

Comment:
I can provide a ES2015 modularized version of lodash, exporting all it's utility methods as functions. That could be loaded and used easily:

```
import { last, sortBy } from '../external/lodash.js'
```

Would that be ok?